File: JAVA/XadrezTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class XadrezTest {

	@Test
	public void testTableColor() throws Exception {
		//Test 1
		int want = 1;
		int got = Xadrez.tableColor(8, 8);
		assertEquals(want, got);
		//Test 2
		want = 1;
		got = Xadrez.tableColor(5, 91);
		assertEquals(want, got);
		//Test 3
		want = 0;
		got = Xadrez.tableColor(6, 9);
		assertEquals(want, got);
	}

}

File: JS/Xadrez.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./Xadrez.js')

TEST('Starting xadrez test...', (t) => {
    //Test 1
    t.assert(OBJ.tableColor(8, 8) === 1, "Expect: 8 8 = 1")
    //Test 2
    t.assert(OBJ.tableColor(5, 91) === 1, "Expect: 5 91 = 1")
    //Test 3
    t.assert(OBJ.tableColor(6, 9) === 0, "Expect: 6 9 = 0")

    t.end()
})
